src/assets/scss/components/_testimonials.scss

```diff
@@ -1,10 +1,26 @@
-        #testimonials {
-            p {
-                margin-bottom: 1em
-            }
-
-            p span {
-                color: _palette(accent2);
-                font-weight: 400;
-            }
-        }
\ No newline at end of file
+#testimonials {
+  p {
+    margin-bottom: 1em;
+  }
+
+  p span {
+    color: _palette(accent2);
+    font-weight: 400;
+  }
+
+  .two {
```

Comment:
above just updates the indentation courtesy of Prettier, new additions to this file are from this line onward